aws/snapshotRDSDatabase.sh

```diff
@@ -97,7 +97,7 @@ status_file="$(getTopTempDir)/snapshot_rds_status.txt"
 # Ensure we are in the right place
 checkInSegmentDirectory
 
-DB_INSTANCE_IDENTIFIER="${PRODUCT}-${SEGMENT}-${TIER}-${COMPONENT}"
+DB_INSTANCE_IDENTIFIER="${PRODUCT}-${ENVIRONMENT}-${TIER}-${COMPONENT}"
```

Comment:
We might have add a bit of logic here. 
With the latest changes we actually now include SEGMENT and ENVIRONMENT in the name, but if the segment is "default" then its not included. 

Would you mind adding that in?

Reply:
@roleyfoley fixed